src/components/GagesBarChartAnimation.vue

```diff
@@ -2659,43 +2662,1465 @@
     <caption id="svgcaption">
       {{ text.caption }}
     </caption>
-    <Infographic />
-    <div>
-      <p>{{ text.svgText }}</p>
-    </div>
     <div id="textWrapArea">
       <div id="gageImageContainer">
-        <div id="gageImage">
+        <!-- <div id="gageImage">
```

Comment:
Maybe this '<!-- ' is not needed

Reply:
good catch

Reply:
I suppose this could be deleted but since we aren't confirmed that we are using the infographic I'll jsut leave the image here for now.